Bottom: 7f1114e51061b97f65c172a8624f5351c84ab866
Top:    7f296bbbae193e5f3bf72998badab5dd92a22c64
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2016-03-06 17:37:16 +0000

Refresh of play-with-quad-edge-data-STEP

---

diff --git a/src/hid/step/object3d_step.c b/src/hid/step/object3d_step.c
index 56a000e..7d1283c 100644
--- a/src/hid/step/object3d_step.c
+++ b/src/hid/step/object3d_step.c
@@ -1,5 +1,4 @@
 #include <stdio.h>
-//#include <stdlib.h>
 #include <stdbool.h>
 
 #include <glib.h>
@@ -10,9 +9,7 @@
 #include "hid/common/contour3d.h"
 #include "hid/common/appearance.h"
 #include "hid/common/face3d.h"
-//#include "hid/common/edge3d.h"
 #include "hid/common/object3d.h"
-//#include "polygon.h"
 #include "data.h"
 
 #include "object3d_step.h"
